### PR TITLE
feat: Add quorum type to space settings

### PR DESCRIPTION
**Pull request by ChaituVR**
Related to https://github.com/snapshot-labs/snapshot/issues/4600

- [x] Waiting for hub changes to merge https://github.com/snapshot-labs/snapshot-hub/pull/827
 
### Summary

Add a new field `quorumType` to space settings
- Default to `default`, or can be `optimistic`
- if value is `default` we store empty string on backend

Untitled

## How to test: 
Make sure to change quorum type to default or optimistic

### CODE REVIEW

File: src/components/SettingsVotingBlock.vue [hunk 1]
```diff
@@ -29,6 +29,25 @@ const { form, validationErrors } = useFormSpaceSettings(props.context);
           block
         />
 
+        <TuneListbox
+          v-model="form.voting.quorumType"
+          placeholder="Default"
+          :label="$t(`settings.quorumType.label`)"
```
Comment (samuveth) [inline src/components/SettingsVotingBlock.vue:35]: We don't want to use i18n anymore, can you hardcode the text?

Comment (samuveth) [reply src/components/SettingsVotingBlock.vue:35]: For label and hint

File: src/composables/useFormSpaceSettings.ts [hunk 2]
```diff
@@ -153,6 +156,7 @@ export function useFormSpaceSettings(
       period: formData.voting.period || undefined,
       type: formData.voting.type || undefined,
       quorum: formData.voting?.quorum || undefined,
+      quorumType: formData.voting.quorumType || undefined,
```
Comment (samuveth) [inline src/composables/useFormSpaceSettings.ts:159]: Shouldn't this default to `default`?
```suggestion
      quorumType: formData.voting.quorumType || 'default',
```

Comment (samuveth) [reply src/composables/useFormSpaceSettings.ts:159]: As it is pruned on line 96 anyways

File: src/helpers/interfaces.ts [hunk 3]
```diff
@@ -179,6 +179,7 @@ export interface ExtendedSpace {
     hideAbstain: boolean;
     period: number | null;
     quorum: number | null;
+    quorumType: string;
```
Comment (wa0x6e) [inline src/helpers/interfaces.ts:182]:
```suggestion
    quorumType: 'default' | 'optimistic';
```

**Review by wa0x6e — COMMENTED** [review]
Since this property is always either `default` or `optimistic`

**Review by samuveth — APPROVED** [review]
tACK

**Pull request merged**